addons/storysource/src/StoryPanel.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from '@storybook/components';
 
 import { SourceBlock, LocationsMap } from '@storybook/source-loader';
+import { Story } from '@storybook/api/dist/lib/stories';
```

Comment:
Should this interface/type be exported from `@storybook/api`?

Reply:
Yeah probably